dupa.txt

```diff
@@ -1 +0,0 @@
-vfdgfdgfdxgds
```

Comment:
asdasdasda

Reply:
teraz  moge tu odpowiedziec

Reply:
Bo teraz jest to komentarz taki jak powinienem dodać, wcześniej dałęm źle